assembly/src/parsers/ast/nodes.rs

```diff
@@ -208,6 +208,7 @@ pub enum Instruction {
     CDropW,
 
     // ----- input / output operations --------------------------------------------------------
+    PushConstants(Vec<Felt>),
```

Comment:
Any reason to reord that? If we should follow a specific order that is not the commonly used alphabetic order, then maybe its better to add a comment to the section.

Reply:
We use this order in most places (i.e., docs): the `push` instruction is listed as the first instruction in the "Input/Output" section.